deploy/operator.yaml

```diff
@@ -1,22 +1,22 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: kogito-cloud-operator
+  name: kogito-operator
 spec:
   replicas: 1
   strategy:
     type: Recreate
   selector:
     matchLabels:
-      name: kogito-cloud-operator
+      name: kogito-operator
   template:
     metadata:
       labels:
-        name: kogito-cloud-operator
+        name: kogito-operator
     spec:
-      serviceAccountName: kogito-cloud-operator
+      serviceAccountName: kogito-operator
       containers:
-        - name: kogito-cloud-operator
+        - name: kogito-operator
           image: quay.io/kiegroup/kogito-cloud-operator:0.6.0
           command:
             - kogito-cloud-operator
```

Comment:
Won't change since the build script generates the final binary with the name of the repository, plus won't have a major change in the image.